### ISSUE TITLE
Default `public/.htaccess` redirects www prefixed FQDN to naked domain

### ISSUE THREAD

**Issue by JedMeister** [ladybirdweb/faveo-helpdesk#8289]
- Faveo Version : v2.0.3 & development
- PHP version : 8.2 (installed from Debian repos)
- Database Driver & Version : MariaDB v10.11.6 (installed from Debian repos)
- Server specification : Apache 2.4 on Debian 12/Bookworm

#### Description:

If the default Faveo Helpdesk `public/.htaccess` file is enabled, then it will redirect a `www` FQDN subdomain to naked domain.

E.g. `www.example.com` will redirect to `example.com`.

I imagine that this is unexpected behaviour for most users - it certainly was for me!

As something of an aside, the default `.htaccess` files still use Apache 2.2 access controls to deny access - see "Access control" section of [Apache 2.2. -> 2.4 docs](https://httpd.apache.org/docs/2.4/upgrading.html#run-time).

#### Steps To Reproduce:

In apache conf, enable `.htaccess` file:

```
[snip]
<Directory /var/www/faveo-helpdesk/public/>
        AllowOverride All        
        Require all granted
</Directory>
[/snip]
```

 and use a subdomain starting with `www.` - see above example.

#### Downloaded from
- [ ] master-branch

- [ ] development-branch

- [x] release-tag

**Comment by Neustradamus** [comment]
@JedMeister: Why to change the redirection rule from www to non-www?

It is important: Drop WWW:
- https://dropwww.com/
- https://dropwww.com/why
- https://dropwww.com/how
- https://dropwww.com/seo

**Comment by Neustradamus** [comment]
@JedMeister: Have you seen my previous comment?

**Comment by JedMeister** [comment]
Yes @Neustradamus - sorry I started responding long ago but got sidetracked and never circled back...

I'm not fundamentally opposed to the idea of "dropping www" but I'm not convinced that the arguments put forward on that site are all legit - at the very least most of them are heavily opinionated rather than "factual".

The "best" way is often subjective or at least has a subjective aspect. Unless there is a _really_ good reason why (e.g. security) my preference is to educate and encourage people to do things the way I think is "best". If what they are doing is not actually doing any harm (which IMO is the case with a 'www' subdomain) why not let them then make an informed decision?

I get the point of not using www but I'd personally rather not railroad others into having to not use it. Besides as a general rule these days I imagine that most sites would be configuring 'www' as a CNAME to the bare domain anyway, making a default redirect redundant.

The reason why I opened this issue (and PR) was simply because I wasted a ton of time troubleshooting why faveo was "broken". I explicitly set a 'www' subdomain and there was no record for the bare domain (I was using 'www.example.com' set in my hosts file) and it took me a while to track down why I kept getting redirected to the real 'example.com'. For better or worse I explicitly chose a www subdomain and didn't expect faveo to be ignoring what I had configured and was using. Perhaps someone else in a similar position who was less persistent and/or knowledgeable might have just given up thinking that faveo _was_ broken?

I think having the redirect in the default `.htaccess` file is a great idea - perhaps even link to that site in a comment? But IMO it should be commented out by default. YMMV...

Anyway, sorry for the rant...
